### ISSUE TITLE
Not recognising 2048

### ISSUE THREAD

**Issue by JamesD-SA** [anpho/2048-in-Cascades#4]
Running latest version on BlackBerry Q10.  Every second or third time I make 2048, the program does not recognise the completion, but just continues waiting for moves.

**Comment by anphorea** [comment]
well that's weired,
Logic is:

if you achieved 2048 at first time, it will show a "YOU WIN" dialog,
choose "Continue", this dialog dismiss, and will not show again ( even if you make a new 2048 tile later )

**Comment by JamesD-SA** [comment]
No, that is not what is happening.  Sometimes – 1 in 2, or 1 in 3 times, when I make 2048 (for the first time), no “you win” dialogue pops up and the game simply continues.  I have not tried to continue to see if anything happens if I make a second 2048 tile.

 

The other times, when the 2048 tile is created, the expected “you win” message pops up.

 

I’m playing on a BlackBerry Q10 using the latest OS 10.2.1.3062.

 

Regards

James

**Comment by anphorea** [comment]
Could you please try the latest version of 2048 from blackberry beta zone and tell me if this problem still exists ?

**Comment by JamesD-SA** [comment]
Hi Merrick

 

Sorry, it took me a day or 2 to get around to looking for it.  Cannot find 2048 on the beta zone, where it does not seem to be listed.  I have version 2.0.15.724 installed on my device.  I’ve encountered the problem several times, most recently about 2 weeks ago – before the latest version was installed, admittedly.

 

Regards

James
